File: Programmers/Level1/Pgm42748.java
package pgm42748;
import java.util.*;
import java.util.stream.Collectors;

public class Solution {
	public static int[] solution(int[] array, int[][] commands) {
		int n = commands.length;
		List<Integer> listA = new ArrayList<Integer>();
		
		for (int i = 0; i < n; i++) {
			int[] arr1 = Arrays.copyOfRange(array, commands[i][0] - 1, commands[i][1]);
			Arrays.sort(arr1);
			System.out.println(Arrays.toString(arr1));
			int k = arr1[commands[i][2] - 1];
			listA.add(k);
			System.out.println("k°ª: " + k);
		
		}
		int[] answer = listA.stream().mapToInt(i->i).toArray();
		System.out.println(Arrays.toString(answer));
		return answer;
	}
}

File: Programmers/Level1/Python/pgm42748.py
# 프로그래머스 - K번째 수 (42748)

def solution(array, commands):
    size = len(commands)
    ans = [0 for i in range(size)]
    for i in range(size):
        arr = array[commands[i][0]-1 : commands[i][1]]
        arr.sort()
        ans[i] = arr[commands[i][2] - 1]
    return ans
